modules/openapi-generator/src/main/resources/go-server/controller-api.mustache

```diff
@@ -90,14 +90,14 @@ func (c *{{classname}}Controller) {{nickname}}(w http.ResponseWriter, r *http.Re
 	{{#allParams}}
 	{{#isPathParam}}
 	{{#isLong}}
-	{{paramName}}Param, err := parseInt64Parameter({{#routers}}{{#mux}}params["{{baseName}}"]{{/mux}}{{#chi}}chi.URLParam(r, "{{baseName}}"){{/chi}}{{/routers}}, {{required}})
+	{{paramName}}Param, err := parseInt64Parameter({{#routers}}{{#mux}}params["{{baseName}}"]{{/mux}}{{#chi}}chi.URLParam(r, "{{baseName}}"){{/chi}}{{/routers}}, {{required}}, {{#minimum}}GetInt64Ptr({{minimum}}){{/minimum}}{{^minimum}}nil{{/minimum}}, {{#maximum}}GetInt64Ptr({{maximum}}){{/maximum}}{{^maximum}}nil{{/maximum}}, {{#defaultValue}}GetInt64Ptr({{defaultValue}}){{/defaultValue}}{{^defaultValue}}nil{{/defaultValue}})
```

Comment:
instead of using nil all over, and requiring ptr. you can use functional options pattern